### PR TITLE
added enrichPubliccodeFromGithubUrl service handler and command

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

**Review by bbrands02 — APPROVED** [review]

File: ActionHandler/GithubFindPubliccodeHandler.php [hunk 1]
```diff
@@ -1,51 +0,0 @@
-<?php
```
Comment (rjzondervan) [inline ActionHandler/GithubFindPubliccodeHandler.php:1]: Please remove this handler also from the InstallationService

File: Command/EnrichPublicodeFromGithubUrlCommand.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Command to execute the EnrichPublicodeFromGithubUrlCommand.
+ */
+class EnrichPublicodeFromGithubUrlCommand extends Command
```
Comment (rjzondervan) [inline Command/EnrichPublicodeFromGithubUrlCommand.php]: Publicode => Publiccode

File: Command/EnrichPublicodeFromGithubUrlCommand.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{
+    protected static $defaultName = 'opencatalogi:enrichPublicodeFromGithubUrl:execute';
```
Comment (rjzondervan) [inline Command/EnrichPublicodeFromGithubUrlCommand.php]: Publicode => Publiccode

**Review by rjzondervan — CHANGES_REQUESTED** [review]
The removed handler was not removed from the InstallationService, which renders errors when running the installation of the bundle.

Further only two typos

**Review by rjzondervan — APPROVED** [review]
Good work!

**Pull request merged**